### PR TITLE
[CPU] LRN node migration on nGraph

### PR DESCRIPTION
**Pull request by mandrono**

### CODE REVIEW

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_lrn_node.cpp [hunk 1]
```diff
@@ -1,58 +1,120 @@
...
+            return false;
+        }
+
+        const auto isSupportedAxes = [](const std::vector<int64_t> &axes, const size_t &dataRank) {
+            if (axes.size() == 1 && axes[0] == 1) {
+                return true;
+            } else {
+                std::vector<bool> norm(dataRank, false);
+                for (auto &axis : axes) {
+                    if (axis < 0) {
```
Comment (dmitry-gorokhov) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_lrn_node.cpp:38]: Seems like we also need to check that `axis < dataRank`

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/lrn.cpp [hunk 2]
```diff
@@ -11,20 +11,76 @@
...
+                                           ::testing::Values(InferenceEngine::Precision::UNSPECIFIED),
+                                           ::testing::Values(std::vector<size_t>({10, 10, 3, 8})),
+                                           ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                        LrnLayerTest::getTestCaseName);
+
+} // namespace LRN4D
+
+namespace LRN5D {
+
+const std::vector<std::vector<int64_t>> axes = {{1}, {2, 3, 4}};
```
Comment (dmitry-gorokhov) [inline inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/lrn.cpp:81]: Does it makes sense to cover cases with non-default axes order like: 3,4,2?

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_lrn_node.cpp (hunk 1 above)
Comment (mandrono) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_lrn_node.cpp]: added check

File: inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/lrn.cpp (hunk 2 above)
Comment (mandrono) [reply inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/lrn.cpp]: expanded

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**